app/models/admin.rb

```diff
@@ -9,6 +9,7 @@ class Admin < ApplicationRecord
          :validatable,
          password_length: ArchiveConfig.ADMIN_PASSWORD_LENGTH_MIN..ArchiveConfig.ADMIN_PASSWORD_LENGTH_MAX,
          reset_password_within: ArchiveConfig.DAYS_UNTIL_ADMIN_RESET_PASSWORD_LINK_EXPIRES.days,
+         reset_password_keys: [:email, :login],
```

Comment:
Would this require both keys to be provided for a reset?

Reply:
Oops, sorry, that wasn't supposed to be here -- I was playing yesterday and forgot to check it out when I was done.